onnxruntime/core/session/custom_ops.cc

```diff
@@ -994,19 +1040,31 @@ common::Status CreateCustomRegistry(gsl::span<OrtCustomOpDomain* const> op_domai
       }
     }
 
+    std::unordered_map<std::string, std::vector<const OrtCustomOp*>> domain_kernels;
+    for (const auto* op : domain->custom_ops_) {
+      // define kernel
+      auto it = domain_kernels.find(op->GetName(op));
+      if (it == domain_kernels.end()) {
+        domain_kernels[op->GetName(op)] = {op};
+      } else {
+        domain_kernels[op->GetName(op)].push_back(op);
+      }
+    }
+
+    for (auto name_op : domain_kernels) {
```

Comment:
Should be able to do this to simplify:

`for (auto& [name, op] : domain_kernels) {`

https://en.cppreference.com/w/cpp/language/structured_binding

`ops` would be a slightly better name given it's a collection.

Reply:
sure, i did not know about this syntax.

Reply:
done